code/obj/item/clothing/suits.dm

```diff
@@ -799,8 +801,17 @@
 		src.cape = FALSE
 		block_vision = !src.eyeholes
 		src.UpdateIcon()
+		src.update_examine()
 		desc = "A linen sheet used to cover yourself while you sleep. Preferably on a bed."
 
+	proc/update_examine()
+		if(src.cape)
+			src.hides_from_examine = null
+		else if(src.eyeholes)
+			src.hides_from_examine = (C_UNIFORM|C_GLOVES|C_SHOES|C_EARS)
```

Comment:
```suggestion
			src.hides_from_examine &= ~(C_GLASSES & C_MASK)
```

Reply:
already tried doing this and it doesn't work

Reply:
Wouldn't it be `&= ~(C_GLASSES | C_MASK)`?

Reply:
Oh yeah oops

Reply:
yeah thats what I had and it didn't work